mkdocs-base.yml

```diff
@@ -119,32 +119,34 @@ extra:
 
 # Common navigation for percona.com, render.com and PDF
 nav:
-  - PMM overview: index.md
-  - Setting up:
-      - setting-up/index.md
-      - Server:
-          - setting-up/server/index.md
-          - setting-up/server/network.md
-          - setting-up/server/docker.md
-          - setting-up/server/podman.md
-          - setting-up/server/helm.md
-          - setting-up/server/virtual-appliance.md
-          - setting-up/server/aws.md
-          - setting-up/server/easy-install.md
-          - setting-up/server/dbaas.md
-      - Client:
-          - setting-up/client/index.md
-          - setting-up/client/mysql.md
-          - setting-up/client/mongodb.md
-          - setting-up/client/postgresql.md
-          - setting-up/client/proxysql.md
-          - setting-up/client/aws.md
-          - setting-up/client/azure.md
-          - setting-up/client/google.md
-          - setting-up/client/linux.md
-          - setting-up/client/external.md
-          - setting-up/client/haproxy.md
-          - setting-up/client/remote.md
+  -  Overview: index.md
+  - PMM fundamentals: fundamentals.md
+  - PMM architecture: architecture.md
+  - Plan your PMM installation: 
+      - plan-pmm-installation/install.md
+      - plan-pmm-installation/resource.md
+      - plan-pmm-installation/network_and_firewall.md
+      - plan-pmm-installation/hardware.md
+      - plan-pmm-installation/ports.md
+  - Install PMM:
+      - install-pmm/index.md
+      - Install PMM server:
+          - install-pmm/install-pmm-server/index.md
```

Comment:
Unindent the - install-pmm/ line two spaces to line up with the first letter (I) of Install PMM.